Sming/third-party/.patches/axtls-8266.patch

```diff
@@ -114,13 +114,10 @@ diff -Nuar a/replacements/mem.c b/replacements/mem.c
 +
 +void ax_port_free(void* ptr) {
 +    free(ptr);
-+    uint32_t *p = (uint32_t*) ptr;
-+    size_t size = p[-3];
-+    if (size >= 1024)
-+        DEBUG_TLS_MEM_PRINT("free %d, left %d\r\n", p[-3], system_get_free_heap_size());
++    DEBUG_TLS_MEM_PRINT("free %d, left %d\r\n", p[-3], system_get_free_heap_size());
```

Comment:
Looking at the [patch file](https://github.com/ADiea/Sming/blob/c12c6bd84d3e316ce5966d0648dca2bc881e2dfa/Sming/third-party/.patches/axtls-8266.patch) I see

```C
+void ax_port_free(void* ptr) {
+    free(ptr);
+    DEBUG_TLS_MEM_PRINT("free %d, left %d\r\n", p[-3], system_get_free_heap_size());
+}
```

In that function `p` is used but it is not declared. So, if I am not totally mistaken, this code should not compile at all. May be you meant to replace `p` with `ptr` casted to uint32_t?